schema/DoctrineMigrations/Version20231218161807.php

```diff
@@ -0,0 +1,107 @@
+                                        `bodyType`
+                                        ) VALUES (
+                                            \'IvozProvider Notifications\',
+                                            \'no-reply@ivozprovider.com\',
+                                            \'IvozProvider Access Credentials\', 
+                                            \'Below are the access credentials for the administration portal of Ivoz Provider:\n User: ${USER}\n Password: ${PASSWORD} \n\', 
+                                            (SELECT id FROM NotificationTemplates WHERE type=\'accessCredentials\'),
+                                            2, 
+                                            \'text/plain\'
+                                        )'
+        );
+
+        $this->addSql('INSERT INTO `NotificationTemplatesContents` (
+                                        `fromName`,
+                                        `fromAddress`,
+                                        `subject`,
+                                        `body`,
+                                        `notificationTemplateId`,
+                                        `languageId`,
+                                        `bodyType`
+                                        ) VALUES (
+                                            \'Notificaciones IvozProvider\',
+                                            \'no-reply@ivozprovider.com\',
+                                            \'Credenciales de acceso IvozProvider\', 
+                                            \'A continuación se le indican las credenciales de acceso para el portal de administración de Ivoz Provider:\n Usuario: ${USER}\n Contraseña: ${PASSWORD} \n\', 
+                                            (SELECT id FROM NotificationTemplates WHERE type=\'accessCredentials\'),
+                                            3, 
+                                            \'text/plain\'
+                                        )'
+        );
+
+        $this->addSql('INSERT INTO `NotificationTemplatesContents` (
+                                        `fromName`,
+                                        `fromAddress`,
+                                        `subject`,
+                                        `body`,
+                                        `notificationTemplateId`,
+                                        `languageId`,
+                                        `bodyType`
+                                        ) VALUES (
+                                            \'IvozProvider Notifications\',
+                                            \'no-reply@ivozprovider.com\',
+                                            \'IvozProvider Access Credentials\', 
+                                            \'Below are the access credentials for the administration portal of Ivoz Provider:\n User: ${USER}\n Password: ${PASSWORD} \n\', 
```

Comment:
\'Below are the access credentials for the administration portal of Ivoz Provider:\n\n\n User: ${USER}\n Password: ${PASSWORD} \n\nBest Regards,\nIvozProvider\n\',